README.md

```diff
@@ -41,8 +41,6 @@ After you're finished please remove all the comments and instructions!
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] You did a fantastic job filling all the required sections on your README file :+1:, but as a general requirement it should also have Key features, a live demo (Here you can write 'Not available'), Future Features and contributing :100:. Each of them should be filled whit the required information